### PR TITLE
add hosting application

### PR DESCRIPTION
**Pull request by mcanoy**
- add hosting application

### CODE REVIEW

File: bootstrap/core/lodestar-apps/lodestar-hostiing.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
```
Comment (oybed) [inline bootstrap/core/lodestar-apps/lodestar-hostiing.yaml:1]: filename has an extra `i` - won't match the `kustomization.yaml` entry

**Review by oybed — CHANGES_REQUESTED** [review]
see inline comment

File: bootstrap/core/lodestar-apps/lodestar-hostiing.yaml (hunk 1 above)
Comment (mcanoy) [reply bootstrap/core/lodestar-apps/lodestar-hostiing.yaml]: renamed

**Review by oybed — APPROVED** [review]
LGTM

**Pull request merged**